packages/pn-personagiuridica-webapp/src/components/Deleghe/ConfirmationModal.tsx

```diff
@@ -32,67 +31,69 @@ export default function ConfirmationModal({
   const isMobile = useIsMobile();
 
   return (
-    <Dialog
+    <PnDialog
```

Comment:
Try to rework the modal using all the dialog components such as PnDialogActions, PnDialogTitle and so on